src/sst/core/elemLoader.cc

```diff
@@ -57,68 +57,6 @@ preload_symbols(void)
 
```

Comment:
Probably should take this opertunity to remove 
```cpp
// README: This is only set if we are not building any elements
// in the split up build this is now default so must always be called.
```
I assume this is a hold over from the monorepo days.

Reply:
In fact if `preload_symbols` entire body is dead code lets just remove it?